app_personal_blog/src/ui/card/card-post/CardPost.module.css

```diff
@@ -0,0 +1,29 @@
+.item {
+   background-color: white;
+   border-radius: 15px;
+   border: none;
+   padding: 20px;
+   text-align: left;
+   cursor: pointer;
+ }
+ 
+ .item_img {
```

Comment:
`item` prefix is not necessary, you can use `wrapper` name for top level style & `title`, `text` etc for other classes